classes/services/PKPNavigationMenuService.inc.php

```diff
@@ -370,6 +368,8 @@ public function loadMenuTree(&$navigationMenu) {
 	 * @return array Hierarchical array of menu items
```

Comment:
This function doesn't appear to return anything -- either fix the code self-documentation, or better yet, use a normal `return` convention instead of the reference (unless there's a reason to do it this way).

Reply:
Fixed